src/main/kotlin/frc/robot/lib/sysid/README.md

```diff
@@ -0,0 +1,193 @@
+# SysIdCommand Factory
+
+`SysIdCommand` is a Kotlin-based helper for characterizing FRC subsystems using WPILib's SysId framework. It simplifies the creation of dynamic and quasistatic tests for any subsystem that implements the `SysIdable` interface and extends `SubsystemBase`.
+
+---
+## Usage
+
+### Implement `SysIdable`
+
+To use `SysIdCommand`, your subsystem must implement the `SysIdable` interface, which requires a method to apply voltage to the mechanism.
+
+```kotlin
+class Wrist : SubsystemBase(), SysIdable {
+    override fun setVoltage(voltage: Voltage) {
+        // Apply voltage to motors
+    }
+}
+````
+
+### Create and Configure a `SysIdCommand`
+
+Use the `sysId()` extension function to generate a `SysIdCommand` and configure it with forward and backward routines.
+
+```kotlin
+val sysIdCommand = wrist
+    .sysId()
+    .withForwardRoutineConfig(/* ... */)
+    .withBackwardRoutineConfig(/* ... */)
+    .command()
+```
+---
+
+## Examples
+
+### Example #1 - Wrist
+This example demonstrates how to implement SysIdable for a mechanism using the Universal TalonFX motor. It defines voltage control for system identification and includes additional functionality for position control using Motion Magic. The configuration is fully parameterized in `WristConstants.kt`, showcasing how to integrate tuning parameters, gearing ratios, and current limits.
```

Comment:
It's great code, but IMO it's too long -- you don't need to show a full example of a subsystem class. Just show what's relevant, along with `wrist.sysId()` usage.